docker/venice-controller/multi-dc-configs/dc-parent.venice.controller.properties

```diff
@@ -0,0 +1,67 @@
+#Mon Oct 10 16:07:33 PDT 2022
+local.region.name=dc-parent-0
+admin.port=5555
+admin.secure.port=5556
+zookeeper.address=zookeeper.dc-parent.venicedb.io\:2181
+kafka.zk.address=zookeeper.dc-parent.venicedb.io\:2181
+kafka.bootstrap.servers=kafka.dc-parent.venicedb.io\:9092
+child.data.center.kafka.zk.dc-parent-0=zookeeper.dc-parent.venicedb.io\:2181
+child.data.center.kafka.zk.dc-0=zookeeper.dc-0.venicedb.io\:2181
+child.data.center.kafka.zk.dc-1=zookeeper.dc-1.venicedb.io\:2181
+child.data.center.kafka.url.dc-parent-0=kafka.dc-parent.venicedb.io\:9092
+child.data.center.kafka.url.dc-0=kafka.dc-0.venicedb.io\:9092
+child.data.center.kafka.url.dc-1=kafka.dc-1.venicedb.io\:9092
+child.cluster.url.dc-1=http\://venice-controller.dc-1.venicedb.io\:5555
```

Comment:
Should we check this version in and later update it to the network based datacenters? or should we explore that now itself?
Reason for this question: It would provide a better view of data centers isolated by networks and we can bring the entire dc up/down by bringing down the networks, etc

Reply:
Yes that's the plan but it's not required for getting started setup

Reply:
I prefer static configs for now